<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>451</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_17">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;2048&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>260</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>260</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
background-color:rgb(163, 163, 163);
border:2px groove gray;
border-radius:10px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label11">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label12">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label13">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label14">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label21">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label22">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label23">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label24">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label31">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label32">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label33">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label34">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label41">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label42">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label43">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label44">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>55</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
margin:2px;
border:2px groove gray;
border-radius:10px;
background-color:rgb(170, 170, 127);
}</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>156</width>
          <height>260</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>156</width>
          <height>260</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
background-color:rgb(220, 220, 165);
border:2px groove gray;
border-radius:10px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonUp">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:5px gray;
border-radius:20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/icons/Up.png</normaloff>:/icons/icons/Up.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Up</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButtonLeft">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:5px gray;
border-radius:20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/icons/Left.png</normaloff>:/icons/icons/Left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Left</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButtonRight">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:5px gray;
border-radius:20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/icons/Right.png</normaloff>:/icons/icons/Right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Right</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButtonDown">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:5px gray;
border-radius:20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/icons/Down.png</normaloff>:/icons/icons/Down.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Down</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButtonRestart">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:5px gray;
border-radius:20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/icons/Restart.png</normaloff>:/icons/icons/Restart.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="shortcut">
              <string>F1</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonExit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:5px gray;
border-radius:20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/icons/Exit.png</normaloff>:/icons/icons/Exit.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="shortcut">
              <string>F2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Panel</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
